2a_model.R

```diff
@@ -189,14 +189,14 @@ p2a_targets_list <- list(
         list(model_id = "0_baseline_LSTM",
              snakefile_dir = "0_baseline_LSTM",
              config_path = stringr::str_remove(p2a_config_baseline_LSTM_yml, "2a_model/src/models/")),
-        # the 1_ models use the same model and therefore the same Snakefile
-        # as the 0_baseline_LSTM run
-        list(model_id = "1_metab_multitask",
-             snakefile_dir = "0_baseline_LSTM",
-             config_path = stringr::str_remove(p2a_config_metab_multitask_yml, "2a_model/src/models/")),
-        list(model_id = "1a_multitask_do_gpp_er",
-             snakefile_dir = "0_baseline_LSTM",
-             config_path = stringr::str_remove(p2a_config_1a_metab_multitask_yml, "2a_model/src/models/")),
+         #the 1_ models use the same model and therefore the same Snakefile
+         #as the 0_baseline_LSTM run
+        #list(model_id = "1_metab_multitask",
+             #snakefile_dir = "0_baseline_LSTM",
+             #config_path = stringr::str_remove(p2a_config_metab_multitask_yml, "2a_model/src/models/")),
+        #list(model_id = "1a_multitask_do_gpp_er",
+             #snakefile_dir = "0_baseline_LSTM",
+             #config_path = stringr::str_remove(p2a_config_1a_metab_multitask_yml, "2a_model/src/models/")),
```

Comment:
I commented these lines out since we didn't train new model weights for these models. 

This undoes 0dbac60e45e7e63bc90c2d422a7c0bcff8d93ef9. @galengorski  - was there a reason you uncommented the 1 and 1a models?